src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/CurrentNode.java

```diff
@@ -0,0 +1,22 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import static edu.greenblitz.tobyDetermined.RobotMap.TelescopicArm.PresetPositions.*;
+import static edu.greenblitz.tobyDetermined.RobotMap.TelescopicArm.PresetPositions;
+
+public class CurrentNode {
+    private static PresetPositions currentNode;
+
+    public static PresetPositions getCurrentNode() {
+        if (currentNode == null)
+            currentNode = INTAKE_GRAB_CONE_POSITION;
```

Comment:
if INTAKE_GRAB_CONE_POSITION is the default position, perhaps save it in preset positions in a DEFAULT_POSITION (  = INTAKE_GRAB_CONE_POSITION; ) so it will be easier to change throughout the code and more informative🙊​